docs/using-the-jdbc-driver/using-plugins/UsingTheLimitlessConnectionPlugin.md

```diff
@@ -0,0 +1,42 @@
+# Using the Limitless Connection Plugin
+
+## What is Amazon Aurora Limitless Database?
+
+Amazon Aurora Limitless Database allows users to horizontally scale their Aurora Databases, handling millions of write transactions per second and managing petabytes of data. To learn more, please [read Amazon Aurora Limitless documentation](// TODO).
```

Comment:
Can you add here whether it is a new type of database or a new type of settings within Aurora?